docs/freqtrade-101.md

```diff
@@ -0,0 +1,106 @@
+# Freqtrade Strategies 101: A Quick Start for Strategy Development
+
+For the purposes of this quick start, we are assuming you are familiar with the basics of trading.
+
+## Required Knowledge
+
+A strategy in Freqtrade is a Python class that defines the logic for buying and selling cryptocurrency `assets`.
+
+Assets are defined as `pairs`, which represent the `coin` and the `stake`. The coin is the asset you are trading using another currency as the stake.
+
+Data is supplied by the exchange in the form of `candles`, which are made up of a six values: `date`, `open`, `high`, `low`, `close` and `volume`.
+
+`Technical analysis` functions analyse the candle data using various computational and statistical formulae, and produce secondary values called `indicators`.
+
+Indicators are analysed on the asset pair candles to generate `signals`.
+
+Signals are turned into `orders` on a cryptocurrency `exchange`, i.e. `trades`.
+
+We use the terms `entry` and `exit` instead of `buying` and `selling` because Freqtrade supports both `long` and `short` trades.
+
+- long:
+  - You buy the coin based on a stake, e.g. buying the coin BTC using USDT as your stake, and you make a profit by selling the coin at a higher rate than you paid for. Profits are made in long trades by the coin value going up versus the stake.
+- short
+  - You borrow capital from from the exchange in the form of the coin, and you pay back the stake value of the coin later. Profits are made in short trades by the coin value going down versus the stake (you pay the loan off at a lower rate).
+
+For simplicity, here we will focus on spot (long) trades only.
+
+## Structure of a Basic Strategy
+
+### Main dataframe
+
+Freqtrade strategies use a tabular data structure with rows and columns known as a `dataframe` to generate signals to enter and exit trades.
+
+Freqtrade dataframes are organised by pair, and are indexed by the `date` column, e.g. `2024-06-31 12:00`.
+
+The next 5 columns represent the `open`, `high`, `low`, `close` and `volume` (OHLCV) data.
+
+### Populate indicator values
+The `populate_indicators` function adds columns to the dataframe that represent the technical analysis indicator values.
+
+Examples of common indicators include Relative Strength Index, Bollinger Bands, Money Flow Index, Moving Average, and Average True Range.
+
+Columns are added to the dataframe by calling technical analysis functions, e.g. ta-lib's RSI function `ta.RSI()`, and assigning them to a column name, e.g. `rsi`
```

Comment:
Is it worth hinting at the fact that not all indicators (also depending on the library) return a series - so this approach may need to be adjusted depending on the library / indicator?
Could also be a case of "no, let's keep it simple" ;)

Reply:
Yeah I'd prefer simplicity for a quick start.